client/src/components/ElementEditor/AddNewButton.js

```diff
@@ -43,8 +43,8 @@ class AddNewButton extends Component {
           target={buttonAttributes.id}
           isOpen={this.state.popoverOpen}
           elementTypes={elementTypes}
-          baseAddHref={baseAddHref}
           toggle={this.toggle}
+          elementalAreaId={elementalAreaId}
```

Comment:
The `insertAfterElement` prop is required but not set here. Additionally I think we want the default location for new blocks to be at the top so it should be `'0'`?

Reply:
Not required, but could have a default. @clarkepaul - can you please confirm whether new blocks should be added at first, or last?

Reply:
Ideal ta, I'll set a defaultProps... err, property :)

Reply:
Went to passing in `0` explicitly instead, because setting a prop to `undefined` is difficult, and I'm unsure the impact `null` will have on the GraphQL query, since it's a valid type (but unexpected by the mutation).

Reply:
Done, no auto resolve as it's the next line down